src/tools/recipe2plan-cli.ts

```diff
@@ -11,13 +11,12 @@ import minimist from 'minimist';
 import fs from 'fs';
 import path from 'path';
 import {Runtime} from '../runtime/runtime.js';
-import {recipe2plan} from './recipe2plan.js';
-import {Flags} from '../runtime/flags.js';
+import {recipe2plan, OutputFormat} from './recipe2plan.js';
 
 const opts = minimist(process.argv.slice(2), {
-  string: ['outdir', 'outfile'],
+  string: ['outdir', 'outfile', 'format', 'recipe'],
   alias: {d: 'outdir', f: 'outfile'},
```

Comment:
Can you update / add aliases for `format` and `recipe`?

Reply:
I think `outfile` should map to `-o`, and `format` to `-f`. 
Though, `outfile`--> `-f` is the convention for a few scripts

Reply:
Given that this program is just invoked from bazel, I don't think we need to spend that much time giving it convenient short arguments. And as you say, `-f` suggests both "format" and "file", so it's a bit confusing.

Reply:
That's why I did not add aliases - they are not obvious I don't think they add much values as it's powered by Bazel.